### PR TITLE
SUS-3369 Quick stats - replaced events with rollup_wiki_user_events

### PR DESCRIPTION
**Pull request by jcellary**
https://wikia-inc.atlassian.net/browse/SUS-3369

### CODE REVIEW

File: extensions/wikia/AdminDashboard/QuickStatsController.class.php [hunk 1]
```diff
@@ -56,14 +56,15 @@ public function getDailyEdits (Array &$stats, $cityID) {
...
-			->WHERE('event_date')->BETWEEN("{$week} 00:00:00", "{$today} 23:59:59")
-				->AND_('wiki_id')->EQUAL_TO($cityID)
+		$db = wfGetDB(DB_SLAVE, array(), $this->wg->DWStatsDB);
+		(new WikiaSQL())
+			->SELECT("date_format(time_id, '%Y-%m-%d')")->AS_('date')
+			->SUM('edits')->AS_('cnt')
+			->FROM('rollup_wiki_user_events')
+			->WHERE('time_id')->BETWEEN("{$week} 00:00:00", "{$today} 00:00:00")
+			->AND_('wiki_id')->EQUAL_TO($cityID)
+			->AND_('period_id')->EQUAL_TO('1')
```
Comment (macbre) [inline extensions/wikia/AdminDashboard/QuickStatsController.class.php]: Let's use const from `DataMartService::PERIOD_ID_DAILY`

File: extensions/wikia/AdminDashboard/QuickStatsController.class.php [hunk 2]
```diff
@@ -56,14 +56,15 @@ public function getDailyEdits (Array &$stats, $cityID) {
...
-		(new WikiaSQL())->skipIf(empty($this->wg->StatsDBEnabled))
-			->SELECT("date_format(event_date, '%Y-%m-%d')")->AS_('date')
-				->COUNT(0)->AS_('cnt')
-			->FROM('events')
-			->WHERE('event_date')->BETWEEN("{$week} 00:00:00", "{$today} 23:59:59")
-				->AND_('wiki_id')->EQUAL_TO($cityID)
+		$db = wfGetDB(DB_SLAVE, array(), $this->wg->DWStatsDB);
+		(new WikiaSQL())
+			->SELECT("date_format(time_id, '%Y-%m-%d')")->AS_('date')
+			->SUM('edits')->AS_('cnt')
```
Comment (macbre) [inline extensions/wikia/AdminDashboard/QuickStatsController.class.php:63]: Shouldn't we sum edits and creates ?

File: extensions/wikia/AdminDashboard/QuickStatsController.class.php (hunk 1 above)
Comment (jcellary) [reply extensions/wikia/AdminDashboard/QuickStatsController.class.php]: :+1:

File: extensions/wikia/AdminDashboard/QuickStatsController.class.php (hunk 2 above)
Comment (jcellary) [reply extensions/wikia/AdminDashboard/QuickStatsController.class.php:63]: You are genius!  I did as you said (but skipped deletes and undeletes, is that fine?)

**Comment by jcellary** [comment]
@macbre I fixed what you suggested, please let me know if it's ok now

**Review by mszabo-wikia — APPROVED** [review]

**Pull request merged**